Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -275,15 +275,16 @@ public void EvaluateActions()
                     }
                 ),
             };
+
             Block<DumbAction> blockIdx2 = MineNext(blockIdx1, blockIdx2Txs);
             pairs = blockIdx2
                 .EvaluateActionsPerTx(dirty1.GetValueOrDefault)
                 .ToImmutableArray();
             expectations = new[]
             {
-                (0, 0, new[] { "A,D", "B", "C", null, null }),
-                (1, 0, new[] { "A,D", "B", "C", "E", null }),
-                (2, 0, new[] { "A,D", "B", "C", "E", "RecordRehearsal:False" }),
+                (0, 0, new[] { "A,D", "B", "C", null, null }, _fx.TxFixture.PrivateKey1.PublicKey.ToAddress()),
```

Comment:
This can be shortened like:

```suggestion
                (0, 0, new[] { "A,D", "B", "C", null, null }, _fx.TxFixture.Address1),
```

Reply:
Fixed. (cf5eedbe757b312358635dec47e565ae0b46e671)